api/src/main/java/graphql/nadel/schema/NadelDirectives.java

```diff
@@ -51,8 +51,15 @@
     static final DirectiveDefinition RENAMED_DIRECTIVE_DEFINITION;
     static final DirectiveDefinition HYDRATED_DIRECTIVE_DEFINITION;
     static final InputObjectTypeDefinition NADEL_HYDRATION_ARGUMENT_DEFINITION;
+    static final DirectiveDefinition DYNAMIC_SERVICE_DIRECTIVE_DEFINITION;
```

Comment:
I believe we haven't transitioned yet to using directives. But I added the new directive here so it's ready when we fully transition.